src/java/org/apache/cassandra/tools/Util.java

```diff
@@ -49,11 +49,12 @@
 import org.apache.cassandra.utils.streamhist.TombstoneHistogram;
 
 import static java.lang.String.format;
+import static org.apache.cassandra.config.CassandraRelevantProperties.TEST_UTIL_ALLOW_TOOL_REINIT_FOR_TEST;
 
 @SuppressWarnings("serial")
 public final class Util
 {
-    public static final String ALLOW_TOOL_REINIT_FOR_TEST = Util.class.getName() + "ALLOW_TOOL_REINIT_FOR_TEST"; // Necessary for testing
+    public static final String ALLOW_TOOL_REINIT_FOR_TEST = TEST_UTIL_ALLOW_TOOL_REINIT_FOR_TEST.getKey(); // Necessary for testing
```

Comment:
this became unsed.

Reply:
I would avoid removing public fields since they can cause compilation errors in dependant projects, however, I think we can do remove them here.
Fixed.